apps/meteor/ee/app/livechat-enterprise/server/hooks/beforeRoutingChat.ts

```diff
@@ -1,23 +1,27 @@
-import { type ILivechatDepartment } from '@rocket.chat/core-typings';
+import type { ILivechatInquiryRecord, SelectedAgent, ILivechatDepartment } from '@rocket.chat/core-typings';
 import { LivechatDepartment, LivechatInquiry, LivechatRooms } from '@rocket.chat/models';
 
 import { notifyOnLivechatInquiryChanged } from '../../../../../app/lib/server/lib/notifyListener';
 import { allowAgentSkipQueue } from '../../../../../app/livechat/server/lib/Helper';
 import { saveQueueInquiry } from '../../../../../app/livechat/server/lib/QueueManager';
 import { setDepartmentForGuest } from '../../../../../app/livechat/server/lib/departmentsLib';
+import { beforeRouteChat } from '../../../../../app/livechat/server/lib/hooks';
 import { online } from '../../../../../app/livechat/server/lib/service-status';
 import { settings } from '../../../../../app/settings/server';
-import { callbacks } from '../../../../../lib/callbacks';
 import { cbLogger } from '../lib/logger';
 
-callbacks.add(
-	'livechat.beforeRouteChat',
-	async (inquiry, agent) => {
+beforeRouteChat.patch(
+	async (
+		originalFn: (inquiry: ILivechatInquiryRecord, _agent?: SelectedAgent | null) => Promise<ILivechatInquiryRecord | null | undefined>,
+		inquiry: ILivechatInquiryRecord,
+		agent?: SelectedAgent | null,
+	) => {
+		await originalFn(inquiry, agent);
```

Comment:
Is it ok to ignore the inquiry returned from the original function?

Reply:
On this case, yes, cause we're not using those values anywhere.

If in the future someones add a behavior on CE that modifies the inquiry data being used here, we should update.